.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
 
       conda config --set show_channel_urls true
       conda update --yes conda
-      conda install --yes conda-build=1.20.0 jinja2 anaconda-client
+      conda install --yes conda-build jinja2 anaconda-client
```

Comment:
Note the previous package installed fine on OS X. So we have not tried to remove or hide it.